tools/presto/presto_fixture.ts

```diff
@@ -0,0 +1,68 @@
+import * as presto from "presto-client";
+
+import { execSync } from "child_process";
+import * as dbadapters from "df/api/dbadapters";
+import { prestoExecute } from "df/api/dbadapters/presto";
+import { sleep, sleepUntil } from "df/common/promises";
+import { IHookHandler } from "df/testing";
+
+const USE_CLOUD_BUILD_NETWORK = !!process.env.USE_CLOUD_BUILD_NETWORK;
```

Comment:
can you refactor this, and the other fixture, to use a proper flag (shared in some common location) now that we have support for them in open-source?

Reply:
I added Flags.string(), but I don't really understand where the value gets set